src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1044,6 +1045,31 @@ private bool CheckPropertyValueKind(SyntaxNode node, BoundExpression expr, BindV
             return true;
         }
 
+        private bool IsBadBaseAccess(SyntaxNode node, BoundExpression receiverOpt, MethodSymbol method, DiagnosticBag diagnostics,
+                                     PropertySymbol propertySymbolOpt = null, bool checkOnlyAccessThroughInterface = false)
+        {
+            if (receiverOpt?.Kind == BoundKind.BaseReference)
+            {
+                var baseReference = (BoundBaseReference)receiverOpt;
+
+                if (method.IsImplementableInterfaceMember() &&
+                    baseReference.ExplicitBaseReferenceOpt?.Type.IsInterfaceType() == true)
+                {
+                    if (TypeSymbol.TryFindBaseImplementationInInterface((NamedTypeSymbol)baseReference.ExplicitBaseReferenceOpt.Type, method, diagnostics, node, this) is null)
+                    {
+                        return true;
```

Comment:
>return true [](start = 24, length = 11)

Should an error be reported before returning true?

Reply:
I see that is done in `TryFindBaseImplementationInInterface`

---
In reply to: [255764725](https://github.com/dotnet/roslyn/pull/33281#discussion_r255764725) [](ancestors = 255764725)